serializations.md

```diff
@@ -356,14 +356,15 @@ purging them from the tree.
 
 #### Contract call
 ```
-[ <caller_address>   :: id()
+[ <caller_id>        :: id()
 , <caller_nonce>     :: int()
 , <height>           :: int()
-, <contract_address> :: id()
+, <contract_id>      :: id()
 , <gas_price>        :: int()
 , <gas_used>         :: int()
 , <return_value>     :: binary()
 , <return_type>      :: int()
+, <log>              :: [{<<address>> :: id, [<<topics>> :: binary()], <<data>> :: binary()}]
```

Comment:
I do not see how `{` `}` have meaning in this document - see [notation](https://github.com/aeternity/protocol/blame/75603a9c9040e32502e9bcf7f87817d94dae5c51/serializations.md#L102-L108).